hypothesis-python/tests/cover/test_lookup_py38.py

```diff
@@ -200,6 +200,9 @@ def has_posonly_args(x, /, y):
     pass
 
 
+# dynamic @example applications modify the shared
+# has_posonly_args.hypothesis._given_kwargs.
+@skipif_threading
 def test_example_argument_validation():
     example(y=None)(has_posonly_args)(1)  # Basic case is OK
```

Comment:
seems like a real latent bug that we're not dealing with?  Unclear that we _should_ do anything, but I think I'd want to xfail this.